Run audit infra & format

datlechin opened the pull request:
This PR are run `audit infra` and `npm run format`

WIP

datlechin committed:
Run audit infra & format

datlechin committed:
remove dist

datlechin committed:
wip

datlechin committed:
wip

datlechin committed:
test build

jtojnar commented:
Can you provide exact commands you ran for each commit? This diff is almost unreviewable and malware could be easily sneaked in.

datlechin committed:
run build

datlechin committed:
wip

flarum-bot committed:
Bundled output for commit 4dd38209574d2475c52240ffb85d3a67122cd203
Includes transpiled JS/TS, and Typescript declaration files (typings).

[skip ci]
